test/runtests.jl

```diff
@@ -1,5 +1,24 @@
 using Test, MixedLayerThermoclineDynamics
 
-@test hello("Julia") == "Hello, Julia"
-@test domath(2.0) ≈ 7.0
-@test MixedLayerThermoclineDynamics.compute_square(2.0) ≈ 4.0
+@time @testset "Grid tests" begin
+    include("test_grids.jl")
+
+    nx, ny = 10, 12
+    Lx, Ly = 2.0, 2.4
+    
+    grid1D = Grid1D(nx, 0, Lx)
+
+    @test test_dx(grid1D, Lx/nx)
+    @test test_xF(grid1D, LinRange(0, Lx, nx))
+
+    grid2D = Grid2D(nx, ny, 0, Lx, 0, Ly)
+
+    @test test_dx(grid2D, Lx/nx)
+    @test test_dy(grid2D, Ly/ny)
+    @test test_xF(grid2D, LinRange(dx/2, Lx - dy/2, nx))
```

Comment:
`dy` was never defined... so I doubt this would work

Reply:
I corrected it, but I don't remember how to run these tests locally.

Reply:
Well zoom with me to show you. It'll make your life easier :)

Reply:
Sure, I'll join your link.

Reply:
In summary, from the terminal while in the repository's main directory you open Julia
```
$ julia --project
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.6.1 (2021-04-23)
 _/ |\__'_|_|_|\__'_|  |
|__/                   |

julia> ]

(MixedLayerThermoclineDynamics) pkg> test
```